gnosis/safe/safe_tx.py

```diff
@@ -138,7 +138,7 @@ def safe_version(self) -> str:
             return self.contract.functions.VERSION().call()
 
     @property
-    def safe_tx_hash(self) -> HexBytes:
+    def safe_tx_data(self) -> Dict:
```

Comment:
I think this name is misleading. What do you think renaming the function to `def eip712_structured_data(self)`?